backend/index.js

```diff
@@ -17,13 +17,22 @@ axios
   })
 
 app.use(express.json())
+
 app.get('/cards', (req, res) => {
   res.json(db.getCards())
 })
+
 app.get('/classCards/:className', (req, res) => {
   res.json(db.getCardsByClass(req.query.className))
 })
 
+app.post('/addDeck', (req, res, next) => {
```

Comment:
You should avoid naming endpoints like this (`/appdeck`). Something along the lines of:
`POST`/`PUT` on `/deck/{deckId}/cards` seems more intuitive to me.
Please validate query param and request body data and handle errors.